pkg/token/server.go

```diff
@@ -30,8 +30,76 @@ import (
 const (
 	DOMAIN_HEADER = "X-Athenz-Domain"
 	ROLE_HEADER   = "X-Athenz-Role"
+	roleToken     = "role token"
+	accessToken   = "access token"
```

Comment:
[type.go](https://github.com/AthenZ/k8s-athenz-sia/blob/68f070a9bd5ef192e2759e77378b4cfb8fbf7fce/pkg/token/type.go#L23-L24)の内容を使うのでも良いかと思いました。[参考](https://github.com/AthenZ/k8s-athenz-sia/blob/68f070a9bd5ef192e2759e77378b4cfb8fbf7fce/pkg/token/daemon.go#L237-L244)

Reply:
handled in https://github.com/AthenZ/k8s-athenz-sia/pull/89/commits/a0f95909e7c049f4874d19e559c9169938a335cd

Since `d.tokenType&mACCESS_TOKEN != 0` is too hard to read & used multiple times in the same function, I've defined variables in the beginning.

Reply:
fixed again in https://github.com/AthenZ/k8s-athenz-sia/pull/89/commits/33bb774452da78fcb7252e221158296b699306ea